readme.md

```diff
@@ -330,7 +330,13 @@ tests/behaviour/setup.sh
 yarn testnet
 ```
 
-3. Run the tests:
+3. Remember to compile the project every time some minor changes are applied could be anoying, so, in a separate terminal, start a process that will be watching for files updates:
```

Comment:
All commands here recompile the project, I don't think there is a need to specify the use of yarn dev (at least in this section)

Reply:
Imo the contributing section should start with something like: "Install dependencies <link to the installation section> and run: `yarn dev`". Next it should show how to run the code. Just then it shall proceed with testing information. It would follow the natural implementation workflow